src/products/firefox_android/how_to_localize.md

```diff
@@ -1,35 +1,17 @@
 # Firefox for Android - How to localize
 
-The release cycle for Firefox for Android is the same as Firefox for Desktop, which is generally 6 weeks - but can sometimes be 8 weeks.
+Firefox for Android is on a four week rapid release cycle. This means that usually there is a new release going out every four weeks.
 
-You can see the [detailed schedule for each release here](https://wiki.mozilla.org/RapidRelease/Calendar).
+Within this timeframe, localizers will usually have four weeks to localize and test their strings for a given release: strings land continuously during the first two weeks, and then stop landing for two weeks so that you can complete your work. The deadline in Pontoon is an indicator of when the next localization cut-off date is for the upcoming release.
 
-*Merge day* is when localizations move to the next train - and the last day available to include localization updates to the next Beta version if you work on tools like Pontoon.
+## Localize Firefox for Android in Pontoon
 
-Note: please make sure that you’ve already localized mozilla.org before you start working on Firefox for Android or any other projects.
+* If your locale has never been set up for any android-l10n project yet, you must fill out [this issue template](https://github.com/mozilla-l10n/android-l10n/issues/new?assignees=delphine&labels=&template=new-localization-request.md&title=Add+%5Blocale%5D+to+%5Bproduct%5D) in order to request to add your locale to “Fenix” (code name for Firefox for Android). You will be notified there once your locale is set up on Pontoon and is ready to be localized.
```

Comment:
The locale is not localized ;-)

> and the project is ready to be localized.

The structure of this section looks really confusing, with the disconnected bullet points, and we should probably get rid of a few "once".

![Schermata 2020-10-14 alle 08 45 36](https://user-images.githubusercontent.com/3644868/95953114-acad7a00-0df9-11eb-9dbc-c441fb46b507.png)

Reply:
Good point, how does it look now?